src/view/Dialog.less

```diff
@@ -0,0 +1,43 @@
+@import (reference) '~nav-frontend-core/less/core';
+
+.dialog {
+  background: @navLysGra;
+  display: flex;
+  flex-direction: column;
+}
+
+.dialog-top {
+  margin: 0.5rem;
+
+}
+
+.venter-pa {
+  display: flex;
+  flex-direction: row;
+  margin-top: 0.5rem;
+  border-style: solid;
+  border-width: 0 0 0.2rem 0;
```

Comment:
Foretrekker `border-bottom: 3px solid #333` i disse tilfellene. 
Imho er dette ett av de veldig få tilfellene der man kan bruke `px` istedet for `rem`, og det er lettere å se hva formålet bak stylingen er når man samler det på en linje sånn.